src/shared/components/defaultTooltip/DefaultTooltip.spec.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import DefaultTooltip from './DefaultTooltip';
+import { assert } from 'chai';
+import { render } from 'enzyme';
+import sinon from 'sinon';
+
+describe("DefaultTooltip", ()=>{
+    it("has min-height: initial", ()=>{
+        /*const tooltip = render(<DefaultTooltip visible={true} overlay={<span>HELLO</span>}><span>TARGET</span></DefaultTooltip>);
+        const inner = tooltip.find('div.rc-tooltip-inner');
+        assert.equal(inner.length, 1, "should be exactly one rc-tooltip-inner element");
+        assert.equal(inner.first().css("min-height"), "initial", "min-height: initial");*/
+        // The above test should work, but its failing, it seems, because I'm losing any reference to the actual
+        //  tooltip, because its rendered in the body root and not in the given element. If I print tooltip.html(),
+        //  I only get '<span>TARGET</span>'. I'm not sure what to do about this.
```

Comment:
yeah... basically I was just trying to add a test that covers the bug, for regression